<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="tests.technix.TechnixTests-20220404140947" tests="6" file="tests/technix.py" time="0.006" timestamp="2022-04-04T14:10:00" failures="0" errors="0" skipped="6">
	<testcase classname="tests.technix.TechnixTests" name="test_WHEN_set_current_THEN_get_current_back_correctly" time="0.004" timestamp="2022-04-04T14:10:00" file="tests\technix.py" line="50">
		<skipped type="skip" message="Cannot catch errors in RECSIM"/>
	</testcase>
	<testcase classname="tests.technix.TechnixTests" name="test_WHEN_set_inhibit_to_active_THEN_inhibit_is_active" time="0.000" timestamp="2022-04-04T14:10:00" file="tests\technix.py" line="69">
		<skipped type="skip" message="Cannot catch errors in RECSIM"/>
	</testcase>
	<testcase classname="tests.technix.TechnixTests" name="test_WHEN_set_inhibit_to_idle_THEN_inhibit_is_idle" time="0.001" timestamp="2022-04-04T14:10:00" file="tests\technix.py" line="64">
		<skipped type="skip" message="Cannot catch errors in RECSIM"/>
	</testcase>
	<testcase classname="tests.technix.TechnixTests" name="test_WHEN_set_local_mode_THEN_mode_is_local" time="0.001" timestamp="2022-04-04T14:10:00" file="tests\technix.py" line="54">
		<skipped type="skip" message="Cannot catch errors in RECSIM"/>
	</testcase>
	<testcase classname="tests.technix.TechnixTests" name="test_WHEN_set_remote_mode_THEN_mode_is_remote" time="0.000" timestamp="2022-04-04T14:10:00" file="tests\technix.py" line="59">
		<skipped type="skip" message="Cannot catch errors in RECSIM"/>
	</testcase>
	<testcase classname="tests.technix.TechnixTests" name="test_WHEN_set_voltage_THEN_get_voltage_back_correctly" time="0.000" timestamp="2022-04-04T14:10:00" file="tests\technix.py" line="46">
		<skipped type="skip" message="Cannot catch errors in RECSIM"/>
	</testcase>
</testsuite>
